tests/fixtures/fixtures_estimation.py

```diff
@@ -15,7 +17,7 @@ def estimation_mock(auth_mock):
         },
         "tiling:1": {"tile_width": 768},
     }
-
+    input_tasks: List[Dict[str, Any]]
```

Comment:
Let's move this to the line below, there's no need to declare it without value assignment